src/shared/tasks/sentryconfig/tasksentryconfig.h

```diff
@@ -0,0 +1,27 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef TASKSENTRYCONFIG_H
+#define TASKSENTRYCONFIG_H
+
+#include <QByteArray>
+#include <QObject>
+
+#include "task.h"
+
+class TaskSentryConfig final : public Task {
+  Q_DISABLE_COPY_MOVE(TaskSentryConfig)
+
+ public:
+  TaskSentryConfig();
+  ~TaskSentryConfig();
+
+  void run() override;
+
```

Comment:
Should the DeletePolicy of this task be changed so it is not deleted?

Reply:
Good catch! I've made it rescheudleable, given that this is not time critical :)